src/docs/forms/components/CheckBox/examples/03-DisableCheckbox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import Form, {
+    CheckBox,
+    TextInput
+} from 'doj-react-adminlte/Form';
+import ValueButton from "doj-react-adminlte/ValueButton";
+
+export default class DisableCheckbox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            checkbox1: null,
+            disabled: null
+        };
+    }
+
+    handleChange = (name, value) => {
+        this.setState({
+            [name]: value
+        });
+    };
+
+    handleClick = () => {
+      this.setState({disabled: true})
+    };
+
+    render() {
+        const {checkbox1,disabled} = this.state;
+        return (
+            <div>
+                <Form onChange={this.handleChange}>
+                    <div className="row">
+                        <div className="col-xs-12">
+                            <span className="text-bold">Fruit</span>
+                            <CheckBox label="Apple"
+                                      name={checkbox1}
+                                      checked={checkbox1}
+                                      disabled={disabled}
+                            />
+                            <ValueButton className="clickToDisable"
```

Comment:
Change to a regular `button`.
Side note: `className` prop is different from `name` prop.